blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/UserDefinedEventHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.rackspacecloud.blueflood.inputs.handlers;
+
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.HttpConfig;
+import io.netty.channel.ChannelDuplexHandler;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.timeout.IdleState;
+import io.netty.handler.timeout.IdleStateEvent;
+import io.netty.handler.timeout.IdleStateHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A generic handler to handle user triggered events. Currently this handler only
+ * handles {@link IdleStateEvent} triggered by {@link IdleStateHandler}
+ *
+ */
+public class UserDefinedEventHandler extends ChannelDuplexHandler {
```

Comment:
How about just naming the Handler specificly for what it does? Like maybe CloseOnReadIdleHandler ?